forge-gui/res/cardsfolder/upcoming/overgrown_pest.txt

```diff
@@ -0,0 +1,7 @@
+Name:Overgrown Pest
+ManaCost:2 G
+Types:Creature Pest
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDig | TriggerDescription$ When CARDNAME enters the battlefield, look at the top five cards of your library. You may reveal a land or double-faced card from among them and put that card into your hand. Put the rest on the bottom of your library in a random order.
+SVar:TrigDig:DB$ Dig | DigNum$ 5 | ChangeNum$ 1 | Optional$ True | ForceRevealToController$ True | ChangeValid$ Land,Card.DoubleFaced | RestRandomOrder$ True
```

Comment:
the DoubleFaced property was named quite badly so it's wrong currently / should be refactored for how the other cards use it

Reply:
Ah, I see what you mean – I'll split this to a different PR